src/revlanguage/distributions/phylogenetics/tree/Dist_FBDP.cpp

```diff
@@ -445,7 +445,7 @@ void Dist_FBDP::setConstParameter(const std::string& name, const RevPtr<const Re
     {
         Mu_timeline = var;
     }
-    else if ( name == "PhiTimeline" || name == "PsiTimeline" )
```

Comment:
I think the idea here is that "Phi" and "Psi" are synonyms.  Is "rho" a synonym for phi/psi?

If so, should we keep the "Psi" synonym when adding the "rho" synonym?

Reply:
According to [this post](https://revbayes.slack.com/archives/C08UR8U3X/p1662526895818889), upper-case "Phi" should be the synonym of "rho" (extant sampling fraction), while lower-case "phi" should be the synonym of "psi" (fossil recovery rate). So the synonymy between "Phi" and "Psi" in the original code was likely a mistake – I'm not sure upper-case "Psi" means anything at all. But @afmagee should feel free to correct me if I got that wrong!

Reply:
Ah, that makes sense.  I'll defer to you and @afmagee on this.  It would be interesting to hear what @afmagee says.

Reply:
Well, that's a bug.

We adopted a unified notation for continuous rates and their event-based analogs, lambda/Lambda, mu/Mu, phi/Phi. In the literature, continuous sampling is often psi, while discrete is often rho (then again, so are mass extinctions). So yes, big-P Psi should be rho

Reply:
Good catch!